addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -145,8 +145,8 @@ export class WebsitePreview extends Component {
 
         onWillUnmount(() => {
             this.websiteService.context.showResourceEditor = false;
-            const { pathname, search, hash } = this.iframe.el.contentWindow.location;
-            this.websiteService.lastUrl = `${pathname}${search}${hash}`;
+            const { pathname, search } = this.iframe.el.contentWindow.location;
+            this.websiteService.lastUrl = `${pathname}${search}`;
```

Comment:
I don't think we want to remove the hash here, this one could be legit as we're taking the iframe's location, not the one of the window.